src/main/java/com/arcbees/website/client/application/bees/BeesView.ui.xml

```diff
@@ -131,108 +133,126 @@
                             </div>
                         </div>
 
-                        <div class="{page.style.bee} {page.style.bee3}">
+                        <div class="{page.style.bee} {page.style.bee2}">
                             <div class="{page.style.beeHolder1}">
                                 <div class="{page.style.beeHolder2}">
                                     <g:Image resource="{page.beeChristian}"></g:Image>
                                     <a href="#{BEE_PHILIPPEB}">
-                                        <h3>Philippe Beaudoin</h3>
-                                        <p>Advisor and co-founder</p>
+                                        <h3>Philippe
+                                            <span>Beaudoin</span>
```

Comment:
BTW, the problem is that they must treat those files as "xml" rather than "html", I'm going to experiment a little bit.